library-best-practices/JLBP-12.md

```diff
@@ -0,0 +1,8 @@
+[JLBP-12] Make level of support and level of API stability clear
+----------------------------------------------------------------
+
+- Surface stability should be clearly described.
```

Comment:
Acrtve voice: Clearly state the stability clients can expect from the API surface. For example,

* This API is stable and suitable for production use, aside from packages marked internal. 
* This API is mostly stable except for the com.foo.canonical package which is changing rapidly and should not be relied on.
* This API is beta and will not change unless a bug or design flaw is uncovered.
* This API is in development. Anything may change at any time. Stable products should not depend on it.

Reply:
I will include a variant of this.